integration_tests/ipp/container/Dockerfile

```diff
@@ -1,16 +1,17 @@
 FROM zgrab2_service_base:latest
 
-RUN apt-get update
-RUN apt-get install -y cups
-RUN apt-get install -y cups-pdf
-RUN echo "ServerAlias *" >> /etc/cups/cupsd.conf
-RUN echo "FileDevice yes" >> /etc/cups/cupsd.conf
-RUN sed -i 's/Order allow,deny/Order allow,deny\'$'\n  Allow all/g' /etc/cups/cupsd.conf
-RUN sed -i 's/Listen localhost:631/Listen 0.0.0.0:631/g' /etc/cups/cupsd.conf
+RUN apt-get update && apt-get install -y \
+  cups \
+  cups-pdf
+# TODO: Provide a pre-built cupsd.conf rather than relying on modifying the default config file
```

Comment:
It looks like this is done

Reply:
Oh yeah, that is done. Oops.

Reply:
By the way, you can merge whenever you're ready -- none of the last round of comments were blocking.